django_usaid_clinicfinder/clinicfinder/serializers.py

```diff
@@ -43,5 +44,15 @@ class LookupPointOfInterestSerializer(HyperlinkedModelSerializer):
 
     class Meta:
         model = LookupPointOfInterest
-        fields = ('url', 'search', 'response', 'location', 'created_at', 'updated_at')
+        fields = (
+            'url', 'search', 'response', 'location')
 
+
+class LBSRequestSerializer(HyperlinkedModelSerializer):
+    pointofinterest = LookupPointOfInterestSerializer(
+        many=False, read_only=False)
```

Comment:
How come ``read_only=False`` here? It seems a bit odd to use a POST to an LBSRequest to update a point of interest?